io/disk/fiotest.py

```diff
@@ -425,4 +425,4 @@ def tearDown(self):
             self.delete_raid()
         dmesg.clear_dmesg()
         if self.err_mesg:
-            self.warn("test failed due to following errors %s" % self.err_mesg)
+            self.log.warn("test failed due to following errors %s" % self.err_mesg)
```

Comment:
@PraveenPenguin @Geet2 we will wait on this PR till we get #2606 get merged.
as this has huge many files changed and fiotest.py is one of it.

Reply:
@Naresh-ibm I have seen the https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2606 it doesn't have any changes in the fiotest.py